app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ArticlesController < ApplicationController
+  def show
+    @article = Article.find(params[:id])
```

Comment:
find the article id from the `articles` database through the corresponding `Article` model and pass it to variable `@article`

Reply:
SELECT ID, TITLE, DESCRIPTION
FROM ARTICLES
WHERE ID = params[:id]

Reply:
To be more specific, this query is more of `SELECT * FROM articles WHERE id=params[:id]`